### PR TITLE
Feature/add unit tests tvos

### PR DESCRIPTION
**Pull request by jaelim-ms**

### CODE REVIEW

File: Config/Global.xcconfig [hunk 1]
```diff
@@ -27,6 +27,9 @@ GCC_C_LANGUAGE_STANDARD = gnu99
 ALWAYS_SEARCH_USER_PATHS = NO
 USER_HEADER_SEARCH_PATHS = "$(SRCROOT)/../MobileCenter/MobileCenter"/**
 
+// :Mark: Code signing
+CODE_SIGN_IDENTITY =
```
Comment (TroubleMakerBen) [inline Config/Global.xcconfig]: Just being curious..do you want to move this here?

File: MobileCenter/MobileCenterTests/MSAppDelegateForwarderTests.m [hunk 2]
```diff
@@ -606,7 +610,9 @@ - (void)testDontForwardSelectorsNotToOverrideIfAlreadyImplementedByOriginalDeleg
 }
 #endif
 
-#if !TARGET_OS_OSX
+#if TARGET_OS_IOS
+
+// TODO: tvOS doesn't support application:openURL:sourceApplication:annotation:. Temporaily disable the test.
```
Comment (TroubleMakerBen) [inline MobileCenter/MobileCenterTests/MSAppDelegateForwarderTests.m]: There is no indication that tvOS will ever have a proper "Safari", I don't think we need those around.

File: Config/Global.xcconfig (hunk 1 above)
Comment (jaelim-ms) [inline Config/Global.xcconfig]: We haven't set any code signing configurations for all the projects and just use default settings. However, somehow, all tvOS targets have signing settings unlike iOS and macOS targets so I wanted to set this value.
The reason of having this in Global is that we need to take care of signing soon so we can address everything together.

File: MobileCenter/MobileCenterTests/MSAppDelegateForwarderTests.m (hunk 2 above)
Comment (jaelim-ms) [inline MobileCenter/MobileCenterTests/MSAppDelegateForwarderTests.m]: Removed all of them.

**Pull request merged**